docs/01-app/03-building-your-application/05-styling/02-tailwind-css.mdx

```diff
@@ -48,17 +62,33 @@ const config: Config = {
   },
   plugins: [],
 }
-export default config
 ```
 
-```js filename="tailwind.config.js" switcher
+</AppOnly>
+
+<PagesOnly>
+
+```ts filename="tailwind.config.ts" highlight={5-7} switcher
+import type { Config } from 'tailwindcss'
+
+export default {
+  content: [
+    './pages/**/*.{js,ts,jsx,tsx,mdx}',
```

Comment:
I think we should keep one block that includes app, pages, and components for now.

Still see people who accidentally forgot to update the config and put code in those spots. Especially if anyone has a codebase with code in both Pages / App.

Further, this will be a non-issue with Tailwind v4, so makes sense to keep it as is.

Reply:
Thank you for good review!
I reverted the codes you mentioned and changed the PR desctiption.